ui/pages/routes/routes.component.js

```diff
@@ -566,6 +584,9 @@ export default class Routes extends Component {
         <Modal />
         <Alert visible={this.props.alertOpen} msg={alertMessage} />
         {!this.hideAppHeader() && <AppHeader location={location} />}
+        {this.showConnectionHeader() && (
+          <ConnectionsHeader location={location} />
```

Comment:
We don't want to place this here, otherwise we risk doing it all the time.  I think it should go directly in `ui/components/multichain/pages/connections/connections.js`.

Reply:
Added in [e4725ee](https://github.com/MetaMask/metamask-extension/pull/21158/commits/e4725eecdf5fe9026f355a9da040edf43348775f)